### ISSUE TITLE
Open editing tab after onboarding

### ISSUE THREAD

**Issue by rgoj** [Maptio/maptio#694]
> When the new map first opens, have the edit map panel open and the circle list fully expanded. Circle details panel should be closed

Source: https://docs.google.com/spreadsheets/d/1b5sR4QRbDaIPtE4rLGatkQM48WXHvoG_uyArHA_3Ikg/edit#gid=0&range=D7

Also see related issue #633

**Comment by rgoj** [comment]
We've got a minimum viable version of this implemented in #712 (and currently on staging).

Some caveats:
* The editing panel will now be open, but not just during the first onboarding! It'll now be open every time a user enters the workspace (e.g. when they start the app and view a map they will see the editing panel; once they close it, it'll stay closed as they click through the different views; but if they go to the organisation settings or close the app, once they view a map again, the editing panel will again be open). This can be addressed (e.g. by saving whether the panel was open or close the last time a map was viewed) in about 1-2 hours as part of the next task we've got on the list. Is it worth it or is what we've got now good enough? (I think so)
*  The nodes will start expanded (i.e. as if the "Expand all" button has been clicked) but only for completely new users. If someone has used Maptio before they'll see a single "closed" node and they'll have to click "Expand all" to see all nodes (just as this works now). This is due to a weird way we're saving local state. Can tackle this in 2 hours or so, ensuring that whenever a user visits a map they've not visited before, all the nodes will be open. From then on, their last state will be saved. Is it worth spending this time?
  * Also worth noting that this issue means that for us, when we test out onboarding, all the nodes will appear closed. To see what a new user would see, we can clear `treeState` out of our local storage in dev tools.
* When I test the onboarding flow, the circle details panel actually does start closed for me! This is not something I'm controlling when I was working on trying to control this I was just wading through state management spaghetti... but when I go through the onboarding flow now and the map opens for the first time, the circle details panel isn't open for me. It's just a byproduct of other changes we've made recently and I can't guarantee it'll work every time, but still a bit of a win!
